shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -717,10 +717,24 @@ void updateSemantics(ByteBuffer buffer, String[] strings) {
                     event.setScrollX((int) position);
                     event.setMaxScrollX((int) max);
                 }
+                event.setItemCount(object.scrollChildren);
+                event.setFromIndex(object.scrollIndex);
+                int visibleChildren = object.childrenInHitTestOrder.size() - 1;
+                for (; visibleChildren >= 0; visibleChildren--) {
+                    SemanticsObject child = object.childrenInHitTestOrder.get(visibleChildren);
+                    if (!child.hasFlag(Flag.IS_HIDDEN)) {
```

Comment:
This is assuming that hidden children will only be at the end of the list? I think that assumption is fair, but might be worth to put a comment?